test/unit/Query.test.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-env mocha */
+import { expect } from 'chai';
+import { buildQueryBody } from '../../src/app/search/query';
+
+describe('Query Body Building', () => {
+  const query = { query: { userQuery: "$shakespeare's, ghost; select *", field: 'title' } };
+
+  it('should not contain certain punctuation', () => {
+    const queryBody = buildQueryBody(query);
+    const parsedValueOne = queryBody.queries[0].value;
+    const parsedValueTwo = queryBody.queries[1].value;
+    const parsedValueFour = queryBody.queries[3].value;
+    expect(parsedValueOne.indexOf('$')).to.equal(-1);
+    expect(parsedValueOne.indexOf(',')).to.equal(-1);
+    expect(parsedValueOne.indexOf('\'')).to.not.equal(-1);
+    expect(parsedValueTwo.indexOf(';')).to.equal(-1);
+    expect(parsedValueFour).to.equal('*');
+  });
+
+  it('should return field query objects for each word within the "queries" array.', () => {
+    const queryBody = buildQueryBody(query);
+    expect(queryBody.queries.length).to.equal(4);
```

Comment:
You should also test each word in the queries array is the word you are expecting in order.